Models/models.js

```diff
@@ -20,3 +18,15 @@ exports.selectArticles = (sort_by = "created_at") => {
     return result.rows;
   });
 };
+
+exports.selectArticleById = (article_id) => {
+  const SQL = "SELECT * FROM articles WHERE article_id = $1";
+
+  return db.query(SQL, [article_id]).then((result) => {
+    if (result.rowCount === 0) {
+      return Promise.reject({ msg: "Page not found!", status: 404 });
+    } else {
+      return result.rows;
```

Comment:
because this is just a single object that we're trying to get - you should return `result.rows[0]` to return the object not inside an array